target/linux/bcm53xx/image/Makefile

```diff
@@ -452,4 +452,14 @@ define Device/tplink_archer-c9-v1
 endef
 TARGET_DEVICES += tplink_archer-c9-v1
 
+define Device/iptime_a5004ns
+  DEFICE_VENDOR = ipTIME
+  DEVICE_MODEL := A5004NS
+  DEVICE_PACKAGES := $(B43) $(USB3_PACKAGES)
+  DEVICE_DTS := bcm4708-iptime-a5004ns
```

Comment:
default value for DEVICE_DTS does not work?

Reply:
i tried just removing it and i got an error
`cat /home/sodo/openwrt/build_dir/target-arm_cortex-a9_musl_eabi/linux-bcm53xx_generic/image-.dtb >> /home/sodo/openwrt/build_dir/target-arm_cortex-a9_musl_eabi/linux-bcm53xx_generic/iptime_a5004ns-kernel.bin
cat: /home/sodo/openwrt/build_dir/target-arm_cortex-a9_musl_eabi/linux-bcm53xx_generic/image-.dtb: No such file or directory`
how should i change it?